src/common/hist_util.cu

```diff
@@ -324,9 +335,14 @@ struct GPUSketcher {
 
       // add cuts into sketches
       thrust::copy(cuts_d_.begin(), cuts_d_.end(), cuts_h_.begin());
+#pragma omp parallel for schedule(static)
```

Comment:
Consider doing it in a parallel loop only starting from a certain number of columns.

Reply:
will do